app/services/decisions/income_decision_tree.rb

```diff
@@ -128,38 +148,73 @@ def after_partner_income_benefits
     end
 
     # <- to make it easier to reimplement when we do self-employed
-    def start_employment_journey
+    def start_client_employment_journey # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
       case form_object.employment_status
       when [EmploymentStatus::EMPLOYED.to_s]
-        edit(:income_before_tax)
+        if FeatureFlags.employment_journey.enabled?
+          edit(:income_before_tax)
+        else
+          show(:employed_exit)
+        end
       when [EmploymentStatus::SELF_EMPLOYED.to_s]
         if FeatureFlags.self_employed_journey.enabled?
           edit('/steps/income/business_type', subject: 'client')
         else
           show(:self_employed_exit)
         end
       when [EmploymentStatus::EMPLOYED.to_s, EmploymentStatus::SELF_EMPLOYED.to_s]
-        redirect_to_employer_details(employment)
+        if FeatureFlags.self_employed_journey.enabled?
+          redirect_to_employer_details(client_employment)
+        else
+          show(:employed_exit)
+        end
+      end
+    end
+
+    def start_partner_employment_journey # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
+      case form_object.partner_employment_status
+      when [EmploymentStatus::EMPLOYED.to_s]
+        if FeatureFlags.employment_journey.enabled?
+          edit('/steps/income/partner/employment_income')
+        else
+          show(:employed_exit)
+        end
```

Comment:
I think the [routing here](https://www.figma.com/design/thkvnDBbQbHlSqzt1TE3lk/Crime-Apply?node-id=14214-72385&t=w4rSIOQHnzonFeBv-0) needs amending, but this pr is quite big. Do you want to address in a separate pr maybe?

Reply:
Yes, i will handle in separate PR